csfieldguide/curriculum_guides/content/en/apcsp/sections/todo.md

```diff
@@ -4,99 +4,108 @@ Several learning objectives are not currently covered by the CS Field Guide.
 This "to-do" list includes general computer science topics and suggestions for how we can update the field guide to cover the learning objectives. Different levels of changes are suggested:
 
 - new chapter: some topics are best put into stand-alone new field guide chapters
```

Comment:
Should we capitalise the first word on each bullet point?

Reply:
It's a very informal list that isn't linked to on the main page anyway, so I don't think it matters